.github/actions/ipsec-key-rotate/action.yaml

```diff
@@ -46,11 +46,10 @@ runs:
         if [[ "${{ inputs.ipv6 }}" == "true" ]]; then
           ((exp_nb_keys*=2))
         fi
+        # If running in tunneling mode, then we have twice the amount of states
+        # and keys to handle encrypted overlay traffic.
         if [[ "${{ inputs.tunnel }}" != "disabled" ]]; then
-          ((exp_nb_keys+=4))
-          if [[ "${{ inputs.ipv6 }}" == "true" ]]; then
-           ((exp_nb_keys+=4))
-          fi
+          ((exp_nb_keys*=2))
```

Comment:
We are adding keys for new ESP tunnels using native IPs (VinE), maybe it should be `exp_nb_keys += (node_number - 1) * 2`?

Reply:
Your PR is correct, I thought we don't have extra v6 keys for VinE :no_mouth: